CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [0.43.1] - 2022-02-01
+
 ### Added
 
 - Add `eks/fargate` distribution option for 2-replica StatefulSet (#346)
```

Comment:
Should we avoid technical details in the Changelog?

```suggestion
- AWS EKS Fargate support (#346)
```

Reply:
I think `eks/fargate` is the helpful bit since it's part of the interface, but removed the details.

Reply:
Ok makes sense, but `Add` prefix is not recommended in the `Added` section according to the [Keep a changelog guidelines](https://keepachangelog.com/en/1.0.0/).